tools/taskcluster/execute-firebase-test.sh

```diff
@@ -55,7 +55,11 @@ exitcode=$?
 echo "Downloading artifacts"
 
 mkdir test_artifacts
-./google-cloud-sdk/bin/gsutil ls gs://firefox-tv_test_artifacts | tail -1 | ./google-cloud-sdk/bin/gsutil -m cp -r -I ./test_artifacts
+./google-cloud-sdk/bin/gsutil ls gs://firefox-tv_test_artifacts | tail -1 | ./google-cloud-sdk/bin/gsutil cp -r -I ./test_artifacts > /dev/null 2>&1
 
 # Now exit the script with the exit code from the test run. (Only 0 if all test executions passed)
+if [[ $exitcode -ne 0 ]]; then
+    echo "UI Test(s) have failed, please check the test artifacts folder"
```

Comment:
This is definitely a step up, but I think we can make this more clear.  The test artifacts folder contains a LOT of files, and it takes a while of sifting through to figure out where the actual error is.

When we run tests locally we see a list of executed tests, which those that failed marked in red up at the top.  I think that should be our goal here, too.

I know that JUnit generates HTML docs when it runs these tests.  I saw a bunch of them for our unit tests in the Test Artifacts folder, and I've seen them after running Espresso locally too.  Is there any way we can pull that file into test artifacts along with everything else, then point to it in this log statement?

Reply:
Most of the test artifacts folder contents are from compile/unit test execution, and anything to do with firebase execution is saved separately into the `test_artifacts` folder for easy discretion.

When Firebase executes UI test, it does not produce HTML docs (and many other docs) unlike local execution, because they want you to log into Firebase to see the results in its own UI. This also happens if you submit tests to Firebase from your local computer.  You won't be able to see which tests have failed, because they want us to log into Firebase for that.

It does save an XML doc within `test_artifacts` folder, containing some details of the test result.  So, if we really want to do that, we need to go into the downloaded folder, fetch the xml file, then parse it to filter the entries of failed tests, and print it out on the log.  But that would be a lot of work (which we probably won't have time to do) compared to just going into the specified URL in the log.

Reply:
Would it be possible to log "Test failed. View test results at www.some.firebase.url/mytestrunid"?

Reply:
hmm, I somehow need to save the output of the gcloud command as json, and seek for the line that contains the URL.  this could be easier.  I'll give a day to see whether I can figure it out.  If not, we can raise a separate bug for this so someone with better scripting knowledge can take out.

Reply:
Actually, there was an easier way.  Disable downloading of artifacts.